tests/test_ordered_serialization.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+from schematics.datastructures import OrderedDict
+from schematics.models import Model
+from schematics.types import StringType, IntType
+
+
+def test_serialize_field_orders_one_two_three():
+    class Example(Model):
+        one = StringType()
+        two = IntType()
+        three = StringType()
+
+        class Options:
+            fields_order = ['one', 'two', 'three']
+
+    p1 = Example({'one': 'a', 'two': 2, 'three': '3'})
+    serialized = p1.to_native()
+    comp = OrderedDict()
+    comp['one'] = 'a'
+    comp['two'] = 2
+    comp['three'] = '3'
+    assert serialized == comp
```

Comment:
Sorry for not being verbose. I think the best way to check order is to check for return value of `items` or type of return value.

```
In [107]: serialized = OrderedDict()

In [108]: serialized['one'] = 'a'

In [109]: serialized['two'] = 2

In [110]: serialized['three'] = '3'

In [111]: order = [('one', 'a'), ('two', 2), ('three', '3')]
In [113]: assert serialized.items() == order

In [114]: assert dict(serialized.items()) == order
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
<ipython-input-114-177698dac240> in <module>()
----> 1 assert dict(serialized.items()) == order

AssertionError:

In [115]: assert dict(serialized) == serialized

In [116]:
```

The problem with above approach is if `to_native` returns `dict` still above test will pass, rather than that checking the output of `.items` is fool proof. Depending on the `ordereddict` or `dict` output of items are different.

```
In [117]: serialized.items()
Out[117]: [('one', 'a'), ('two', 2), ('three', '3')]

In [118]: dict(serialized).items()
Out[118]: [('one', 'a'), ('three', '3'), ('two', 2)]

```